[2021-02-23 14:57:14,123] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_subdag_operator.section-1 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:57:15,160] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_subdag_operator.section-1 2021-02-21T00:00:00+00:00 [queued]>
[2021-02-23 14:57:15,162] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:15,164] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:57:15,167] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:15,972] {taskinstance.py:1063} INFO - Executing <Task(SubDagOperator): section-1> on 2021-02-21T00:00:00+00:00
[2021-02-23 14:57:16,164] {standard_task_runner.py:52} INFO - Started process 108 to run task
[2021-02-23 14:57:16,417] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_subdag_operator', 'section-1', '2021-02-21T00:00:00+00:00', '--job-id', '32', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_subdag_operator.py', '--cfg-path', '/tmp/tmpo0frs315', '--error-file', '/tmp/tmpu8ozluh0']
[2021-02-23 14:57:16,448] {standard_task_runner.py:77} INFO - Job 32: Subtask section-1
[2021-02-23 14:57:17,600] {logging_mixin.py:104} INFO - Running <TaskInstance: example_subdag_operator.section-1 2021-02-21T00:00:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:57:19,247] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_subdag_operator
AIRFLOW_CTX_TASK_ID=section-1
AIRFLOW_CTX_EXECUTION_DATE=2021-02-21T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-21T00:00:00+00:00
[2021-02-23 14:57:19,842] {subdag.py:157} INFO - Created DagRun: scheduled__2021-02-21T00:00:00+00:00
[2021-02-23 14:58:22,785] {base.py:245} INFO - Success criteria met. Exiting.
[2021-02-23 14:58:23,311] {subdag.py:171} INFO - Execution finished. State is success
[2021-02-23 14:58:23,932] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_subdag_operator, task_id=section-1, execution_date=20210221T000000, start_date=20210223T145714, end_date=20210223T145823
[2021-02-23 14:58:26,549] {local_task_job.py:188} WARNING - State of this instance has been externally set to success. Terminating instance.
[2021-02-23 14:58:26,720] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 108
[2021-02-23 14:58:26,736] {taskinstance.py:1239} ERROR - Received SIGTERM. Terminating subprocesses.
[2021-02-23 14:58:26,951] {process_utils.py:66} INFO - Process psutil.Process(pid=108, status='terminated', exitcode=1, started='14:57:15') (108) terminated with exit code 1
